source/user-manual/api/configuration.rst

```diff
@@ -5,63 +5,56 @@
 Configuration
 =============
 
-The API will bind to port 55000/tcp by default and requires username and password authentication. The default username and password is "foo" and "bar".
-
-
-Configuration script
---------------------
-
-Run the script ``/var/ossec/api/scripts/configure_api.sh`` to configure the basic settings.
-
-The script supports both unattended and attended configuration. To set the parameters of the unattended configuration use the file ``/var/ossec/api/configuration/preloaded_vars.conf``. This file will be removed after running the script to remove any sensitive information written there.
+By default, the API will bind to port 55000/tcp and use self-signed certificates. Please review the :ref:`Securing API <securing_api>` section for more information on how to protect it.
 
 Configuration file
 ------------------
 
-You can configure certain API settings in the file ``/var/ossec/api/configuration/config.js``: ::
-
-    // Path
-    config.ossec_path = "/var/ossec";
-    // The host to bind the API to.
-    config.host = "0.0.0.0";
-    // TCP Port used by the API.
-    config.port = "55000";
-    // Use HTTP protocol over TLS/SSL. Values: yes, no.
-    config.https = "yes";
-    // Use HTTP authentication. Values: yes, no.
-    config.basic_auth = "yes";
-    //In case the API run behind a proxy server, turn to "yes" this feature. Values: yes, no.
-    config.BehindProxyServer = "no";
-
-Make sure to restart wazuh-api service after editing the configuration file:
-
-  a. For Systemd:
+The API configuration can be found inside ``/var/ossec/api/configuration/api.yaml``. By default, all settings are commented out. To apply a different configuration, uncomment it.
```

Comment:
Add something like `For additional details about the different configuration options available check the Configuration options section` at the end of the paragraph with a link to that section.